certbot-ci/certbot_integration_tests/utils/pebble_ocsp_server.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+"""
+This runnable module interfaces itself with the Pebble management interface in order
+to serve a mock OCSP responder during integration tests against Pebble.
+"""
+import datetime
+import re
+
+import requests
+from dateutil import parser
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography import x509
+from cryptography.x509 import ocsp
```

Comment:
Won't this fail in our nightly tests where the version of `cryptography` installed doesn't have OCSP support?

Reply:
It would, but the situation were it would occur is the oldest integration tests `integration-certbot-oldest` and `integration-nginx-oldest`, and these tests are using Boulder.

However I moved the import of pebble_ocsp_server inside the dedicated method that setup Pebble environment, to not raise a useless `ImportError` on oldest tests when Boulder is used.